kratos/tests/materials_with_subproperties.json

```diff
@@ -0,0 +1,110 @@
+{
+    "properties" : [{
+        "model_part_name" : "Inlets",
+        "properties_id"   : 1,
+        "Material"        : {
+            "constitutive_law" : {
+                "name" : "HyperElastic3DLaw",
+                "todo_name" : "GeneralParallelConstitutiveLaw",
+                "sub_properties_indexes"   : [ 11,  12,  13 ],
+                "combination_factors"      : [0.4, 0.3, 0.3 ],
+                "material_rotation_angles" : [ 0,   45,  90 ],
```

Comment:
material angle means nothing for a 3D law...and i would defefinitely not store it unless it is really needed.
same for other properties...anyway the constructor will be able to parse it if you actually happen to need it.

Reply:
This is just an example, it will be implemented in a future

Reply:
The material angles would allow to define the orientation of each (anisotropic) ply inside a laminate. Only one angle is employed since the lamination is supposed parallel to XY plane.
Additionally, three euler angles will be needed (per element, or per integration point) to define the orientation of the whole laminate in a 3D geometry.
The implementation of both (ply orientation and laminate orientation) is still pendent, and should be discussed to agree its better way of doing it.

Reply:
Yes

Reply:
i still don't think it is a good example...you already have the combination factors to prove your point ... and even then i would store them as a variable rather than in the CL

Reply:
you did not address this comment